index.html

```diff
@@ -4,99 +4,617 @@
+              />
+              <span class="header__logo">THEGEM</span>
             </div>
-            <div class="col l-7">
-              <div class="navigation">
-                <ul class="nav-text">
-                  <li class="nav-text-item">HOME</li>
-                  <li class="nav-text-item">PAGES</li>
-                  <li class="nav-text-item">BLOG</li>
-                  <li class="nav-text-item">PORTFOLIOS</li>
-                  <li class="nav-text-item">SHOP</li>
-                  <li class="nav-text-item">FEATURES</li>
-                  <li class="nav-text-item">ELEMENTS</li>
+            <div class="header__navbar">
+              <div class="header__navbar__menu">
+                <input type="checkbox" id="toggle-menu" class="toggle-menu" />
+                <label for="toggle-menu" class="header__menu">
+                  <i class="fa-solid fa-bars"></i>
+                </label>
+                <ul class="header__navbar__menu__items">
+                  <li class="header__navbar-item">HOME</li>
+                  <li class="header__navbar-item active">PAGES</li>
+                  <li class="header__navbar-item">BLOG</li>
+                  <li class="header__navbar-item">PORTFOLIOS</li>
+                  <li class="header__navbar-item">SHOP</li>
+                  <li class="header__navbar-item">FEATURES</li>
+                  <li class="header__navbar-item">ELEMENTS</li>
                 </ul>
               </div>
-            </div>
-            <div class="col l-2">
+
               <div class="nav-icon">
                 <i>
-                  <img src="./assets/img/america.png" alt="america" />
+                  <img
+                    class="nav-icon__america"
+                    src="./assets/img/america.png"
+                    alt="america"
+                  />
                 </i>
                 <i>
                   <img src="./assets/img/search.png" alt="search" />
                 </i>
                 <i>
-                  <img src="./assets/img/bag.png" alt="bag" />
-                </i>
-                <i>
-                  <img src="./assets/img/icon.png" alt="icon" />
+                  <img
+                    class="nav-icon__bag"
+                    src="./assets/img/bag.png"
+                    alt="bag"
+                  />
                 </i>
+
+                <div class="triangle"></div>
+                <span class="nav-icon-blue">2</span>
               </div>
             </div>
           </div>
         </div>
+      </div>
 
-        <div id="slider">
-          <div class="row no-gutters">
-            <div class="col l-12">
-              <div class="slide--img">
-                <img
-                  class="slide--img__menu"
-                  src="./assets/img/menu.png"
-                  alt="menu img"
-                />
+      <div class="content">
+        <div class="content__slider">
+          <div class="icon-arrow-right">
```

Comment:
```suggestion
          <div class="slider__arrow slider__arrow--left">
```
Viết chung 1 class slider__arrow để style chung cho 2 arrow